state_machine/applications/system-check/main.py

```diff
@@ -76,3 +83,8 @@ async def main_test():
 
 tasko.add_task(main_test(), 1)
 tasko.run()
+
+nvm_reset = input(f"\n\nWould you like to reset non-volatile memory? Select {bold}(y){normal} for yes," +
```

Comment:
I'd rather not have this here - I think it belongs in the nvm test file.

When the NVM test runs, all of the NVM registers are changed. Do they get reset at the end of the test?
Let's just have the NVM test reset all of the NVM as it completes.